app/templates/eventView.html

```diff
@@ -123,5 +123,12 @@ <h3><br>Program Trainings</h3>
                     <button type="submit" class="btn btn-primary" value="{{eventData.id}}" id="rsvpBtn">RSVP</button>
                  {% endif %}
               {% endif %}
+              {% if g.current_user.isCeltsAdmin or isProgramManager %}
+            <div class="d-flex align-items-center">
+                <i class="bi bi-eye-fill me-2"></i>
+                <span class="me-2">Page Views:</span>
```

Comment:
Also, after thinking about it more, "Unique Viewers" would probably be the best descriptor.

Reply:
how about we make it like "Single User Views" because Unique Viewers could be a bit misleading

Reply:
How is it misleading? Unique Viewers seems to be a very accurate descriptor to me. It is a count of all unique users that have viewed the page.

Reply:
haha, I meant that Unique Viewers could refer to certain users and we refer to them as unique

Reply:
Brian and I agree that Unique Viewers is the best description here.